util/imagetools/printers.go

```diff
@@ -229,15 +270,99 @@ func (p *Printer) printManifestList(out io.Writer) error {
 	return w.Flush()
 }
 
+func (p *Printer) printImageConfigs(ics map[string]*ocispecs.Image, out io.Writer) error {
```

Comment:
Not sure about this formatting. We don't even have anything like this for `docker inspect`. Maybe leave this out of release and discuss later.

Reply:
Yes was just because it was "missing" following the fields we currently expose.